### PR TITLE
Remove unused variables from variables.css

### PR DESCRIPTION
**Pull request by jonbell-lot23**
**Background**

I ran a script that finds items in variables.css that aren't used anywhere else in the codebase. Then I went one by one to see if the class could be safely removed. Here's a screenshot of my spreadsheet:

image

And here's the spreadsheet itself:
[variables-alignment.csv](https://github.com/replayio/devtools/files/13733727/variables-alignment.csv)

### CODE REVIEW

File: packages/replay-next/pages/variables.css [hunk 1]
```diff
@@ -63,17 +63,13 @@
   --cool-gray-200: #e5e7eb;
   --cool-gray-300: #d1d5db;
   --cool-gray-400: #9ca3af;
-  --cool-gray-500: #6b7280;
```
Comment (bvaughn) [inline packages/replay-next/pages/variables.css:66]: Why?

File: src/ui/components/shared/ReplayLogo.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ const logoSizes = {
 const logoColors = {
   white: "#FFF",
   fuschia: "#F02D5E",
-  gray: "var(--cool-gray-500)",
+  gray: "#6b7280)",
```
Comment (bvaughn) [inline src/ui/components/shared/ReplayLogo.tsx:12]: Why? Seems like a regression.

**Review by bvaughn — APPROVED** [review]
approve all except `--cool-gray-500`

**Comment by jonbell-lot23** [comment]
Like the other PR, I dealt with the cool-gray-500 thing. (Same change across two branches because I had some PRs stacked up and was trying to avoid conflicts)

**Pull request merged**